examples/interactive_play.py

```diff
@@ -612,12 +659,21 @@ def has_pygame():
         action="store_true",
         help="If specified, does not add the inverse kinematics end-effector control.",
     )
+
+    parser.add_argument(
+        "--use-joint-control",
+        action="store_true",
+        default=False,
+        help="Control joints of articulated agent.",
```

Comment:
Does this work with any articulated agent? I thought interactive play already supports articulated robots.

Reply:
It doesn't, I could instead have a single flag for human, or check if human is in the config file to activate this.

Reply:
I would make this an explicit flag like `use-humanoid-agent`. I think we should revamp the `interactive_play.py` script in the future, so just use the simplest, clearest option.

Reply:
Resolved